app/components/chat/Messages.js

```diff
@@ -10,9 +12,19 @@ export default (props) => {
           { messages.map((message, idx) => {
             return (
               <div key={idx}>
```

Comment:
Using `idx` as the key here is actually bad, especially since we add messages from the top. This will mess up React's reconciliation process and slow down re-rendering. Is there anything unique about each message that we can use instead?